client/ayon_slack/plugins/publish/integrate_slack_api.py

```diff
@@ -368,7 +355,25 @@ def send_message(self, channel, message, publish_files):
             error_str = self._enrich_error(str(e), channel)
             self.log.warning("Not SlackAPI error", exc_info=True)
 
-        return None, []
+    def _upload_attachments(self, publish_files):
+        """Returns list of permalinks to uploaded files"""
+        file_urls = []
+        for published_file in publish_files:
+            with open(published_file, "rb") as f:
+                uploaded_file = self.client.files_upload_v2(
+                    filename=os.path.basename(published_file),
+                    file=f.read()
```

Comment:
Shouldn't this be just?
```suggestion
                    file=f
```

Reply:
It sends the images right now, could try it without `.read()` though.

Reply:
It works even without `read`, I removed it and committed.